packages/components/tests/integration/components/hds/pagination/size-selector-test.js

```diff
@@ -0,0 +1,53 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module(
+  'Integration | Component | hds/pagination/size-selector',
```

Comment:
I would also
- test that a label exists and has the right `for` attribute (since the ID is dynamic, you can see how we did it here with the form elements: https://github.com/hashicorp/design-system/blob/main/packages/components/tests/integration/components/hds/form/select/field-test.js#L75-L99)
(no need to test more the actual select, since it's already tested as "base" HDS form select.